### PR TITLE
MapSelector Size/Position Updated

### PR DESCRIPTION
**Pull request by Verttox**
Changed position to empty space.
Changed size to fit space.
Removed hardcoded SetSize in MapSelectorDialog.cpp
Added wide/tall parameters to DialogMapSelector.res

### CODE REVIEW

File: mp/src/game/client/momentum/ui/MapSelection/MapSelectorDialog.cpp [hunk 1]
```diff
@@ -56,7 +56,7 @@ CMapSelectorDialog::CMapSelectorDialog(VPANEL parent) : Frame(nullptr, "CMapSele
     SetParent(parent);
     SetScheme(scheme()->LoadSchemeFromFile("resource/MapSelectorScheme.res", "MapSelectorScheme"));
     SetProportional(true);
-    SetSize(GetScaledVal(680), GetScaledVal(400));
+    SetPos(vgui::scheme()->GetProportionalScaledValue(235), vgui::scheme()->GetProportionalScaledValue(115));
```
Comment (Gocnak) [inline mp/src/game/client/momentum/ui/MapSelection/MapSelectorDialog.cpp]: Any reason to not use `GetScaledVal` here?

Also, why set the pos here if you moved the size to the res file. Shouldn't the pos also be there as well?

Comment (Verttox) [reply mp/src/game/client/momentum/ui/MapSelection/MapSelectorDialog.cpp]: I didnt see that GetScaleVal was the same thing but neat my bad.
I put setpos in codebecause it works with different aspect ratios so wouldnt need to be changed but there is no reason for it to not be in the .res you are right. 

fixed it up

**Comment by Gocnak** [comment]
Do these changes scale to different aspect ratios and resolutions?

**Comment by Verttox** [comment]
Position scales to resolution and aspect ratio, size only scales to resolution but not to aspect ratio sadly. 
Album with different resolution and aspect ratios: https://imgur.com/a/qxPXBkh

From what i gathered while googleing the issue the getscaledval/>GetProportionalScaledValue dont work with aspect ratio changes and i have not found another solution present in the code already, i assume it would need a new function that detects users aspect ratio and then scales the the getscaledval result to that aspect ratio with some math.

**Comment by Gocnak** [comment]
Fair enough. We do want to be able to scale to aspect ratio as well. Proportional scaling only takes the value in terms of 640x480 (4:3) and scales it to the current resolution.

But have you seen https://developer.valvesoftware.com/wiki/VGUI_Position_and_Size_Flags ? Could help with the sizing thing.

**Comment by Verttox** [comment]
huh.. you are right, it actually works. 

I saw those but for some reason thought that would use screensize and not actual resolution/aspectratio set by the user. anyways i put that in now and it now scales to all the aspect ratios i can test. 

It does give the collums a lot less space on 4:3 obviously so thats maybe something to look into

File: mp/src/game/client/momentum/ui/MapSelection/MapSelectorDialog.cpp [hunk 2]
```diff
@@ -56,7 +56,7 @@ CMapSelectorDialog::CMapSelectorDialog(VPANEL parent) : Frame(nullptr, "CMapSele
     SetParent(parent);
     SetScheme(scheme()->LoadSchemeFromFile("resource/MapSelectorScheme.res", "MapSelectorScheme"));
     SetProportional(true);
-    SetSize(GetScaledVal(680), GetScaledVal(400));
+    SetSize(GetScaledVal(600), GetScaledVal(300));
```
Comment (Gocnak) [inline mp/src/game/client/momentum/ui/MapSelection/MapSelectorDialog.cpp]: So would it be worth doing this still if you were able to figure out how to make the res file stuff work?

Comment (Verttox) [reply mp/src/game/client/momentum/ui/MapSelection/MapSelectorDialog.cpp]: For now yes.

Removing it causes the issue discribed in commit https://github.com/momentum-mod/game/pull/452/commits/ba5aa3e99da51fc0c4035946b1450497bbec24e6.

"Moved size back into MapSelectorDialog.cpp as it broke sizes set for column headers in BaseMapsPage.cpp when set from DialogMapselector.res"

Really not sure why this happens.

**Comment by Gocnak** [comment]
Do me a favor and do:
`git reset HEAD~2`
Recreate your "added 'tall' parameter' commit without the gitignore change
Commit it, and then do:
`git push --force`

We want commits to be clean from both redundancy and vulgarity

File: mp/src/game/client/momentum/ui/MapSelection/MapSelectorDialog.cpp (hunk 2 above)
Comment (Gocnak) [reply mp/src/game/client/momentum/ui/MapSelection/MapSelectorDialog.cpp]: That's fine then

**Pull request merged**